ui/src/components/core/input/input.styles.ts

```diff
@@ -59,8 +59,13 @@ const styles = {
   },
 };
 
-export const Input = styled('input', styles);
+export const InputStyled = styled('input', styles);
 
-export const Textarea = styled('textarea', styles);
+export const InputIconStyled = styled(Icon, {
+  position: 'absolute',
+  left: '$4',
+  top: '0.9375rem',
+  color: 'slate8',
+});

```

Comment:
Should this be a variant of `Icon`?

Reply:
mmm not sure, cause it's only to do this component 
<img width="495" alt="image" src="https://user-images.githubusercontent.com/52458215/217516298-0568e08b-6490-4b8a-a2fc-0db48cb2eb9d.png">